user-manual.md

```diff
@@ -0,0 +1,52 @@
+# What is RateMyCSE?
+RateMyCSE is a course reviewing website specifically for the students of the University of Washington studying computer science. It is a place where you can find everything you need on a CSE class in order to build a balanced schedule. You can read written reviews from students who have taken the class, the average difficulty, workload, and practicality ratings of each class, see tags for classes that have coding, exams, or group projects (work in progress). You can also provide insight for other students by leaving your own reviews and ratings.
+
+# How to install and run RateMyCSE
+RateMyCSE is a free, public website. You can access it by visiting rate-my-cse.vercel.app
```

Comment:
Nitpick: can you turn the link into a clickable link using the '[rate-my-cse.vercel.app](rate-my-cse.vercel.app)' link format

Reply:
Oop woops I thought it was ! Fixed.